src/components/ExperimentDetail.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState } from 'react'
+// import { Link } from 'react-router-dom'
+import { Container, Row, Col, Alert, Card, Progress, CardBody, CardText, FormGroup, Label, Input, CustomInput, DropdownToggle, DropdownMenu, DropdownItem, UncontrolledButtonDropdown, Nav, NavItem, NavLink, TabContent, TabPane } from "reactstrap";
+import "../static/css/Experiments.css";
+import classnames from 'classnames';
+ 
+
+const ExperimentDetail = () => {
+
+    const data = [
+        { id: "1", emojis: "🚀💸", name:"Up Only WBTC" , contract: "OxSalo12345", assetPrice: "1.00", currency: "$", totalAssets: "1000", totalAUM: "10000", baseAsset: "WBTC"},
+      ];
+
+    const [activeTab, setActiveTab] = useState('1');
+
+    const toggle = (tab: any) => {
+        if(activeTab !== tab) setActiveTab(tab);
+    }
+
+    return (
+        <Container className="experiments-container container h-100">
+            <Row className="row h-100 justify-content-center align-items-center">
+                <Col className="experiments-column col-12">
+                <div>
+                    <Card className="card card-detail">
+                    <div>
+                        {data.map(item => (
+                            <div className="title-section"><div className="emoji">{item.emojis}</div>{item.name}
+                            <CustomInput className="switch-label float-right" type="switch" id="approveSwitch" name="approveSwitch" label="Approve Vault" />
+                            
+                            </div>
+                        ))}                        
+                    </div>
+                    <Alert color="warning">
+                        ⚠️ <strong>WARNING</strong> this an experiment of an experiment, of a series of experiments, made by apes and for them.<br></br>Proceed with extreme caution.
+                    </Alert>
+                    <Nav pills justified>
+                        <NavItem>
```

Comment:
This would be a decent place to use `map`.

```
const navItems = ['Details', 'Wallet', 'Contracts'];

...

{navItems.map((navItem: string, navItemKey: number) => {
    const tab = (navItemKey + 1).toString();

    return (
    <NavItem>
        <NavLink
        className={classnames({ active: activeTab === tab })}
        onClick={() => {
            toggle(tab);
        }}
        >
        {navItem}
        </NavLink>
    </NavItem>
    );
})}
```

Reply:
Great ideia, incorporated!